gnoland/website/main.go

```diff
@@ -72,6 +79,38 @@ func handlerFaucet(app gotuna.App) http.Handler {
 	})
 }
 
+func handlerStatusJSON(app gotuna.App) http.Handler {
+	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		var ret struct {
+			ConnectedToGnoland bool    `json:"connected-to-gnoland"`
+			CurrentHeight      int64   `json:"current-height"`
+			Uptime             float64 `json:"uptime-seconds"`
+			// versions
+			// processed txs
+			// active connections
+			Version sdk.Result
+		}
+		ret.Uptime = time.Since(startedAt).Seconds()
+
+		res, err := makeRequest(".app/version", []byte{})
```

Comment:
Don’t know why, but err is nil and res is empty

Reply:
it seems that the version was not configured.
https://github.com/gnolang/gno/blob/15141fdf703a7b4b5ea2b63ba90d66c2eb46856f/gnoland/app.go#L24
missing:
```go
baseApp.SetAppVersion("1.0.0")
```

I have tested it locally, but version is returned in [Response.Value](https://github.com/gnolang/gno/blob/15141fdf703a7b4b5ea2b63ba90d66c2eb46856f/pkgs/sdk/baseapp.go#L419)  instead  `Response.Data`

Reply:
Thanks, it works now!